packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -356,6 +357,9 @@ class FlutterInspectorSettingsDialog extends StatelessWidget {
               gaItem: gac.inspectorHoverEvalMode,
             ),
             const SizedBox(height: denseSpacing),
+            ...dialogSubHeader(theme, 'Default Inspector Details View'),
```

Comment:
Looks like Kenzie wanted this subheading removed (so that the options fall under general)